### ISSUE TITLE
Investigate ViT Approaches

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#130]
Vision Transformers are gaining steam. However, the smallest size networks tend to be much larger than e.g. MobileNet V2; additionally, the datasets tend to be quite large as well. However, this is an area to watch (in conjunction with unsupervised/semi-supervised approaches) to look for advancements.

**Comment by wingman-jr-addon** [comment]
Compact Convolutional Transformers looks like an intriguing hybrid approach. Keras even has a nice bite-sized example for it: https://keras.io/examples/vision/cct/

**Comment by wingman-jr-addon** [comment]
I'd still like to improve the model performance quite a bit, and this seems like a concrete way to start easing into Vision Transformers, so I'm going to label it high to work down the path of evaluating this some more.

**Comment by wingman-jr-addon** [comment]
So my initial research here leads me to believe that I'd need considerably more data to make this approach work well, and the model sizes haven't been optimized down like they have for convolutional networks to the same degree, so no clear path here.
